### PR TITLE
Simplify `package` visibility declaration.

### PR DESCRIPTION
**Pull request by armiol**
The summary of changes:

* explicit `/* package */` declaration has been removed.
* code has been _gently_ reformatted where applicable to adjust spacing after `/* package */` removal.

### CODE REVIEW

**Comment by armiol** [comment]
@alexander-yevsyukov PTAL.

File: server/src/main/java/org/spine3/server/stand/SubscriptionRegistry.java [hunk 1]
```diff
@@ -52,7 +52,7 @@
      * @param subscription the subscription to activate
      * @param callback     the callback to make active
      */
-    /* package */
+
```
Comment (alexander-yevsyukov) [inline server/src/main/java/org/spine3/server/stand/SubscriptionRegistry.java]: The empty line separates Javadoc comment from the method. Please remove.

File: server/src/main/java/org/spine3/server/stand/SubscriptionRegistry.java [hunk 2]
```diff
@@ -66,7 +66,7 @@ synchronized void activate(Subscription subscription, Stand.EntityUpdateCallback
      * @param target the target for a new subscription
      * @return the created subscription
      */
-    /* package */
+
```
Comment (alexander-yevsyukov) [inline server/src/main/java/org/spine3/server/stand/SubscriptionRegistry.java]: The empty line separates Javadoc comment from the method. Please remove.

File: server/src/main/java/org/spine3/server/stand/SubscriptionRegistry.java [hunk 3]
```diff
@@ -94,7 +94,7 @@ synchronized Subscription addSubscription(Target target) {
      *
      * @param subscription the subscription to remove
      */
-    /* package */
+
```
Comment (alexander-yevsyukov) [inline server/src/main/java/org/spine3/server/stand/SubscriptionRegistry.java]: The empty line separates Javadoc comment from the method. Please remove.

File: server/src/main/java/org/spine3/server/stand/SubscriptionRegistry.java [hunk 4]
```diff
@@ -114,7 +114,7 @@ synchronized void removeSubscription(Subscription subscription) {
      * @param type the type to filter by
      * @return the collection of filtered records
      */
-    /* package */
+
```
Comment (alexander-yevsyukov) [inline server/src/main/java/org/spine3/server/stand/SubscriptionRegistry.java]: The empty line separates Javadoc comment from the method. Please remove.

File: server/src/main/java/org/spine3/server/stand/SubscriptionRegistry.java [hunk 5]
```diff
@@ -126,7 +126,7 @@ synchronized void removeSubscription(Subscription subscription) {
      * @param type the type to check records for
      * @return {@code true} if there are records with the given type, {@code false} otherwise
      */
-    /* package */
+
```
Comment (alexander-yevsyukov) [inline server/src/main/java/org/spine3/server/stand/SubscriptionRegistry.java]: The empty line separates Javadoc comment from the method. Please remove.

File: server/src/test/java/org/spine3/server/aggregate/AggregateRepositoryShould.java [hunk 6]
```diff
@@ -399,7 +409,8 @@ protected TestAggregateRepository(BoundedContext boundedContext) {
         /** Needs to be static. */
         private static final Map<CommandId, Command> commandsHandled = newHashMap();
 
-        @SuppressWarnings("PublicConstructorInNonPublicClass") /** It is required to be public. */
+        @SuppressWarnings("PublicConstructorInNonPublicClass")
+        /** It is required to be public. */
```
Comment (alexander-yevsyukov) [inline server/src/test/java/org/spine3/server/aggregate/AggregateRepositoryShould.java]: This shouldn't probably be a Javadoc comment, but a comment for warning suppression, which comes on the same line. The excessive star was an error in the first place.

**Pull request merged**